docs/accessibility/keyboard-navigation.md

```diff
@@ -0,0 +1,55 @@
+# Keyboard Navigation
+
+MakeCode includes features to assist in navigation when the keyboard is used as the primary input method. When a pointing device (mouse or touch input) is not used, or its use is limited, the keyboard assists navigation using these features:
+
+* A hidden menu of shortcuts appears when the user begins to tab into the editor. Help text is displayed for the shortcuts.
+* Navigation into menus and the simulator are standardized.
+
+## Drop-down menu
+
+When navigating through the editor with the keyboard, focus is shown with a blue outline around the interactive element. When a contextual menu opens, the ``Top`` and ``Bottom`` arrow keys navigate into, through, and out of the menu. The ``Enter`` or ``Space`` keys validate the selection. Pressing ``Tab`` or ``Shift + Tab`` moves to the next or previous interactive element outside of the current drop-down menu.
+
+![](/static/images/accessibility-context-menu.png)
+
+## Dialogs
+
+Sometimes MakeCode shows a dialog on top of the editor. When a dialog is displayed, keyboard navigation is restricted to just the elements inside the dialog.
+
+To close a dialog, navigate to the close button (shown as an X at the top right or top left corner) and press ``Enter`` or ``Space``. The ``Escape`` key will dismiss the dialog without having to navigate to the close button.
+
+## Simulator
+
+Currently, the simulator is only partially accessible. We are working hard to make it fully accessible soon. The following are the supported inputs and sensors.
+
+### Buttons
+
+A button press in the simulator is similar to button press elsewhere. A button is activated by pressing either the ``Enter`` or ``Space`` key.
+When the simulator is representing a physical board, the touch related inputs (buttons, switches, capacitive pins) support a long press duration. You can perform touch-like input in the simulator with the  ``Enter`` or ``Space`` key.
+
+### Thermometer, Light level, Noise level and Pins
```

Comment:
microbit specific

Reply:
Remove the first 2 sentences. We should not start apologizing right away.